### ISSUE TITLE
Acceptance Criteria - RL

### ISSUE THREAD

**Issue by rickeylynch** [Christopher-Green424/websiteQAExample1#4]
## Devices
Device name MacBook Pro (Retina, 15-inch, Early 2013)
Processor 2.7 GHz Quad-Core Intel Core i7
System type 64-bit operating system, x64-based processor

## Acceptance Criteria
1. The website should comply with WCAG 2.1 AA ADA standards.
- [ ] - Check if passed.
- Notes: not completely ADA compliant (return to top does not meet ADA standards for example)
2. There is a navbar and footer.
- [ ] - Check if passed.
- Notes: There is a footer but not an actual navbar minus the link to the contact page at the top of the page 
3. All of the links on the website function as intended.
- [ ] - Check if passed.
- Notes: Instagram link at the bottom of the page is non-functional (404 error)
4. The logo of the club is present and readable.
- [x] - Check if passed.
- Notes:
5. The website's footer has at least four working social media links.
- [x] - Check if passed.
- Notes: 3 of the 4 social media links at the bottom of the page work
6. The website has a consistent theme between pages.
- [x] - Check if passed.
- Notes:
7. The website has at least 5 bordered images on it.
- [x] - Check if passed.
- Notes:
8. The website has a contact page with a form and Google Maps embed. 
- [x] - Check if passed.
- Notes:
9. All resources used in the creation of the website are correctly cited and available for use.
- [x] - Check if passed.
- Notes:
